barretenberg/cpp/scripts/strip-wasm.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
-./src/wasi-sdk-20.0/bin/llvm-strip ./build-wasm/bin/barretenberg.wasm
+/opt/wasi-sdk/bin/llvm-strip ./build-wasm/bin/barretenberg.wasm
```

Comment:
Why not rely on llvm strip being in path?

Reply:
It's not in PATH as wasi-sdk bins don't have unique prefixes.
Cross compilers usually might be named like `wasi-wasm-clang++` or someting. Maybe that could be fixed in the wasi build? Then we could add wasi to PATH.